DFC.Digital/DFC.Digital.Services.SendGrid.UnitTests/packages.config

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="FakeItEasy" version="4.5.1" targetFramework="net472" />
+  <package id="FluentAssertions" version="5.6.0" targetFramework="net472" />
+  <package id="MSTest.TestAdapter" version="1.2.1" targetFramework="net472" />
+  <package id="MSTest.TestFramework" version="1.3.2" targetFramework="net472" />
```

Comment:
Why do we need this? Are we not using xUnit?

Reply:
The tests are not running in the test explorer without these packages, so could not check my code coverage etc